.github/workflows/create-scheduled-issues/template/wg-components-office-hours.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const { getGoogleCalendarAddEventLink } = require("../utils");
+
+const descriptionUrl = 'https://go.amp.dev/wg-components-hours-announcement';
+const vcUrl = 'https://go.amp.dev/wg-components-hours';
```

Comment:
@rebeccanthomas If we know the actual URLs, we can use them on a manual run instead of golinks.